### ISSUE TITLE
Chronological view

### ISSUE THREAD

**Issue by neuhaus** [loomchild/sojourner-web#14]
Please add a view of all events sorted by time

**Comment by loomchild** [comment]
You are right. I removed it thinking tha search function will replace it. However today I discovered that it’s still useful to see all events when you don’t know what to do. 

I am sorry for that, will surely fix it for next FOSDEM.

**Comment by neuhaus** [comment]
Thanks. The app worked great for me at FOSDEM2019!

**Comment by loomchild** [comment]
Thank you for coming to FOSDEM and using my little app. I also really appreciate your feedback.
